### PR TITLE
Delete isLockDealBundlePoolId from LockDealBundleProvider contract

### LINKED ISSUE #99: delete extra data
no need to do this check - if we want to add another security, let's check that poolIdToLockDealBundle > 0
https://github.com/The-Poolz/LockDealNFT/blob/8bafa3b6cab03e7f02159fc25bf88510db3b0ec6/contracts/LockDealBundleProvider/LockDealBundleProviderState.sol#L9

### PR DESCRIPTION
**Pull request by 0xSolcroot**
close #99

### CODE REVIEW

File: contracts/LockDealBundleProvider/LockDealBundleProviderModifiers.sol
Comment (Lomet) [inline contracts/LockDealBundleProvider/LockDealBundleProviderModifiers.sol:1]: let's delete this file

Comment (0xSolcroot) [reply contracts/LockDealBundleProvider/LockDealBundleProviderModifiers.sol:1]: yep, deleted

File: contracts/LockDealBundleProvider/LockDealBundleProvider.sol [hunk 1]
```diff
@@ -88,7 +87,12 @@ contract LockDealBundleProvider is
 
     function withdraw(
         uint256 poolId
-    ) public override onlyNFT onlyBundlePoolId(poolId) returns (uint256 withdrawnAmount, bool isFinal) {
+    ) public override onlyNFT returns (uint256 withdrawnAmount, bool isFinal) {
+        require(
+            poolIdToLockDealBundle[poolId].firstSubPoolId > 0,
+            "Pool is not a bundle pool"
+        );
```
Comment (Lomet) [inline contracts/LockDealBundleProvider/LockDealBundleProvider.sol:94]: let's remove this. 
we got the `onlyNFT`
the NFT can call only the ID of the provider.

File: contracts/LockDealBundleProvider/LockDealBundleProvider.sol [hunk 2]
```diff
@@ -109,7 +113,12 @@ contract LockDealBundleProvider is
     ) public onlyProvider {
     }
 
-    function getData(uint256 poolId) public view override onlyBundlePoolId(poolId) returns (IDealProvierEvents.BasePoolInfo memory poolInfo, uint256[] memory params) {
+    function getData(uint256 poolId) public view override returns (IDealProvierEvents.BasePoolInfo memory poolInfo, uint256[] memory params) {
+        require(
+            poolIdToLockDealBundle[poolId].firstSubPoolId > 0,
+            "Pool is not a bundle pool"
+        );
```
Comment (Lomet) [inline contracts/LockDealBundleProvider/LockDealBundleProvider.sol:120]: this is view function. no need to check this.
if it have no data, it will show 0

**Review by Lomet — APPROVED** [review]

**Pull request merged**